### ISSUE TITLE
Support "time" per datapoint for writePoints instead of only one "time" for all in 4.x

### ISSUE THREAD

**Issue by Apollon77** [node-influx/node-influx#201]
Hey,

I know that in 5.0.0 the writePoints will be changed, but to contiue with my current project I would need to have the option to use writePoints with a dedicated "time" property for each data point and not only have one as it is currently.

I think that the patch itself would be easy and can be done backward compatible because I would use the "time" when set with datapoint, and the "time" from first when not provided ...

The issue is to ask if you plan to do a 4.2.3 and would accept that change (before I can bring in a pull request)

**Comment by connor4312** [comment]
Works for me!

**Comment by Apollon77** [comment]
hehe, started looking into it and it seems that I just misunderstood the documentation  ... :-)
Will test later today and maybe no change required at all :-)

**Comment by connor4312** [comment]
ah, yes, sorry, just read the gist of the issue when I woke up... We do currently [have support](https://github.com/node-influx/node-influx/blob/master/index.js#L384) for writing the time in the series. However, if you encounter any bugs with it please do open an issue :)

**Comment by Apollon77** [comment]
Sure I will.
I understood the one sentence about "time in "first object" as if it is only allowed to have in first "datapoint" and will then be reused ... but this was wrong :-)

**Comment by ericvrp** [comment]
Is there an example for writing time per datapoint. 5.0.0-alpha.4 does not work as expected for me.

**Comment by connor4312** [comment]
Here's an example, pulling from the tests:

```js
return influx.writePoints([
  {
    measurement: 'mymeas',
    tags: { my_tag: '1' },
    fields: { myfield: 90 },
    timestamp: new Date(1463683075000),
  },
]);
```

Can you give some more details about what is wrong and post your code?

**Comment by ericvrp** [comment]
Thanks! You are right, this is working! 

I probable made two mistakes. The most important one being that I used 'time' instead of 'timestamp' and the secondly what I probably did wrong is that I provided timestamps in the future with:

> timestamp: new Date(Date.now() + pointsIndex * oneSecond),

When I start with a time in the past instead of Date.now() things work as expected. I will need to give the InfluxDB documentation a better look.
